test/index.test.js

```diff
@@ -1148,7 +1168,8 @@ jobs:
 
             return scm.decorateUrl({
                 scmUri,
-                token: 'mytokenfortesting'
+                token: 'mytokenfortesting',
+                scmContext: 'github.com'
```

Comment:
I think that comment make sense. We fix `data-schema` to add `scmContext` as optional (https://github.com/screwdriver-cd/data-schema/pull/155/commits/ad694c46cc818c7019eac8384a7e0cf30f3e30c9), then reduce a lot of extra logic.